src/Surfnet/StepupMiddleware/ApiBundle/Identity/Projector/AuditLogProjector.php

```diff
@@ -91,6 +100,9 @@ private function applyAuditableEvent(AuditableEvent $event, DomainMessage $domai
             $entry->actorId         = $metadata['actorId'];
             $entry->actorCommonName = $actor->commonName;
         }
+        if (property_exists($auditLogMetadata, 'vettingType') && !is_null($auditLogMetadata->vettingType)) {
```

Comment:
Why not transfer the if to a helper function "hasVettingType"?  Would make it easier to read.

Reply:
In fact, you could just create a helper method to add to the actorCommonname conditionally.  That should also reduce the cyclomatic complexity meaning you can get rid of the warning suppression.

Something like...

```PHP
private function enhanceActorCommonNameWithAuditLog ($entry, $auditLogMetadata) {
  if (property_exists($auditLogMetadata, 'vettingType') && !is_null($auditLogMetadata->vettingType)) {
      return $entry->actorCommonName . $auditLogMetadata->vettingType->auditLog();
  }
  return $entry->actorCommonName
}
```

Reply:
I like this suggestion. Will pickitup